Kunena/Kunena-Forum
# Revert change on config.xml

Pull Request for Issue #9717 
Pull Request for Issue #9698
 
#### Summary of Changes 
#9717 Revert change on config.xml  and change 2 other files
#9698 Change plugin.js and change x.png to x_social.png
#### Testing Instructions
See my comments on issue #9717 

As far as I could test, the changes I made are working.
Change x/plugin.js and renaming image allows to hide x from editor toolbar
